api/src/main/java/org/pmiops/workbench/config/WorkbenchConfigMapper.java

```diff
@@ -77,5 +77,6 @@ default void mapRuntimeImages(WorkbenchConfig source, @MappingTarget ConfigRespo
   @Mapping(target = "rasClientId", source = "config.ras.clientId")
   @Mapping(target = "rasLogoutUrl", source = "config.ras.logoutUrl")
   @Mapping(target = "freeTierBillingAccountId", source = "config.billing.accountId")
+  @Mapping(target = "currentDuccVersions", source = "config.access.currentDuccVersions")
```

Comment:
This is the list of DUCC versions which are acceptable for RT/CT membership.  They are NOT the versions which can be shown in SIGNED mode.